### PR TITLE
add warning for agent-slim

### PR DESCRIPTION
**Pull request by wpalmeri**
as it currently must be run inside k8s

### CODE REVIEW

File: agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml [hunk 1]
```diff
@@ -77,6 +77,8 @@ spec:
           readOnly: true
       containers:
       - name: sysdig-agent
+        # WARNING: the agent-slim release currently must be run within
+        # a kubernetes cluster to properly function
```
Comment (anoop-sysd) [inline agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml]: can we make this more specific? say, "NOTE: agent-slim depends on agent-kmodule so is supported only within Kubernetes".

Comment (anoop-sysd) [reply agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml]: just fyi, agent-kmodule runs as an init container for agent-slim.

Comment (wpalmeri) [reply agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml]: is there something that a customer is expected to do with that information? that feels like an implementation detail

Comment (wpalmeri) [reply agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml]: ah i see what you are getting at. i'll change the comment

**Pull request merged**